packages/styles/scss/components/tableofcontents/_tableofcontents.scss

```diff
@@ -308,7 +308,7 @@ $hover-transition-timing: 95ms;
     > {
       svg {
         position: absolute;
-        fill: currentColor;
+        fill: $text-01;
```

Comment:
Thanks @BrunnoM7 !!! just looked at the specs and it seems like we should be using `$icon-02` here. 
<img width="738" alt="Screen Shot 2021-08-31 at 2 17 20 PM" src="https://user-images.githubusercontent.com/20210594/131570103-50e80135-613b-4783-8dcb-f561b8c2aad7.png">

Reply:
Thanks @ariellalgilmore, I just made a new commit with the right color token.